src/media/Media.php

```diff
@@ -2,31 +2,26 @@
 /**
  * The Media specific functionality.
  *
- * @since   1.0.0
- * @package Eightshift_Boilerplate\Media
+ * @package EightshiftBoilerplate\Media
  */
 
 declare( strict_types=1 );
 
-namespace Eightshift_Boilerplate\Media;
+namespace EightshiftBoilerplate\Media;
 
-use Eightshift_Libs\Media\Media as Libs_Media;
+use EightshiftBoilerplateVendor\EightshiftLibs\Media\Media as MediaMedia;
```

Comment:
`MediaMedia`? That's just lazy naming.

Maybe `LibsMedia` or `CoreMedia` or something like that (suggestions welcomed @infinum/wordpress-team )

Reply:
This is auto added by intelisence